app/models/atmosphere/virtual_machine.rb

```diff
@@ -98,6 +98,8 @@ def uuid
 
     def reboot
       cloud_action(:reboot, :reboot)
+    rescue Excon::Errors::Conflict
+      # ok reboot already in progress
     end
```

Comment:
Would it be possible to check that vm is already being rebooted based on vm model state kept in Atmo DB? I am aware that this state is updated periodically but maybe our system should not allow to reboot a vm that it thinks is being rebooted (or in booting state). What do you think?